lib/surrealist.rb

```diff
@@ -45,20 +46,24 @@ def included(base)
     #   Surrealist.surrealize_collection(User.all)
     #   # => "[{\"name\":\"Nikita\",\"age\":23}, {\"name\":\"Alessandro\",\"age\":24}]"
     #   # For more examples see README
-    def surrealize_collection(collection, camelize: false, include_root: false, include_namespaces: false, root: nil, namespaces_nesting_level: DEFAULT_NESTING_LEVEL) # rubocop:disable Metrics/LineLength
-      unless collection.respond_to?(:each)
-        raise Surrealist::ExceptionRaiser.raise_invalid_collection!
+    def surrealize_collection(collection, camelize: false, include_root: false, include_namespaces: false, root: nil, namespaces_nesting_level: DEFAULT_NESTING_LEVEL, raw: false) # rubocop:disable Metrics/LineLength
+      raise Surrealist::ExceptionRaiser.raise_invalid_collection! unless collection.respond_to?(:each)
+
+      s_c = collection.map do |record|
+        if Helper.surrealist?(record.class)
+          record.build_schema(
+            camelize: camelize,
+            include_root: include_root,
+            include_namespaces: include_namespaces,
+            root: root,
+            namespaces_nesting_level: namespaces_nesting_level,
+          )
+        else
+          record
+        end
       end
 
-      JSON.dump(collection.map do |record|
-        record.build_schema(
-          camelize: camelize,
-          include_root: include_root,
-          include_namespaces: include_namespaces,
-          root: root,
-          namespaces_nesting_level: namespaces_nesting_level,
-        )
-      end)
+      !!raw ? s_c : JSON.dump(s_c) # rubocop:disable Style/DoubleNegation
```

Comment:
Few notes here: 
1. Why not just `raw ?`.
2. What does `s_c` mean? :)

Reply:
1. I can't really remember haha
2. Surrealized_Collection, maybe need a better var name cause that'd be too long